drivers/watchdog/Kconfig.cc13xx_cc26xx

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2021 Florin Stancu <niflostancu@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+DT_COMPAT_TI_CC13XX_CC26XX_WDG := ti,cc13xx-cc26xx-watchdog
+
+config WDT_CC13XX_CC26XX
+	bool "Watchdog Driver for CC13xx / CC26xx family of MCUs"
+	default $(dt_compat_enabled,$(DT_COMPAT_TI_CC13XX_CC26XX_WDG))
```

Comment:
How about `DT_HAS_<COMPAT>_ENABLED` and drop both below `depends`, see e.g. bdff3e76e5853fdc9dce2e72858dd9936b90d8fa